### DISCUSSION TITLE
Separate datetime syntax from language setting

### DISCUSSION THREAD

**Discussion by Tayruu** [stoatchat/discussions#558] [Feature Suggestions]
One problem I ran into with Discord, and appears to be repeated here, is that between UK English and US English, there are different elements that I prefer. Namely - I want 12-hour time (which is in US English), but DD/MM/YYYY (which is in UK English). Or heck, even YYYY-MM-DD would be good.

To resolve this issue, I feel users should be able to setup their own date-time syntax. If it's possible, Revolt could also try to read the OS's setting for this, as I have my machines set up with custom settings for things like YYYY-MM-DD, though I don't know how open Windows/Mac/Linux distros et al are about region locale settings.

I understand that not all users will be familiar with using date-time syntax fields, so such a setting could perhaps be an optional override over the language setting, but it would be a start.

**Comment by ThomasJRyan** [comment]
Seems to be implemented as of now
